api/endpoints.go

```diff
@@ -132,6 +132,89 @@ func handleAccountLogout(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+func handleAddFriend(w http.ResponseWriter, r *http.Request) {
+	formErr := r.ParseForm()
+	if formErr != nil {
+		httpError(w, r, fmt.Errorf("failed to parse request form: %s", formErr), http.StatusBadRequest)
+		return
+	}
+
+	uuid, err := uuidFromRequest(r)
+	if err != nil {
+		httpError(w, r, err, http.StatusBadRequest)
+		return
+	}
+
+	success, err := db.AddFriend(uuid, r.Form.Get("username"))
```

Comment:
IMO it would be nice to have a different name than `username`, cause it is ambiguous

Reply:
I don't really see how it's ambiguous, `/account` implies it's already linked to a player account so we're not retrieving the friends of an account if that's what you mean.
I also don't really like having `/account/friends?friend=..`